packages/contracts/src/artifacts/LibOrder.json

```diff
@@ -30,9 +29,8 @@
-            "updated_at": 1518051461266
+            "unlinked_binary": "0x6060604052341561000f57600080fd5b6103ae8061001e6000396000f300606060405260043610610041576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff168063cfc4d0ec14610046575b600080fd5b341561005157600080fd5b6100b66004808060a001906005806020026040519081016040528092919082600560200280828437820191505050505091908060c0019060068060200260405190810160405280929190826006602002808284378201915050505050919050506100d4565b60405180826000191660001916815260200191505060405180910390f35b6000308360006005811015156100e657fe5b60200201518460016005811015156100fa57fe5b602002015185600260058110151561010e57fe5b602002015186600360058110151561012257fe5b602002015187600460058110151561013657fe5b602002015187600060068110151561014a57fe5b602002015188600160068110151561015e57fe5b602002015189600260068110151561017257fe5b60200201518a600360068110151561018657fe5b60200201518b600460068110151561019a57fe5b60200201518c60056006811015156101ae57fe5b6020020151604051808d73ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff166c010000000000000000000000000281526014018c73ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff166c010000000000000000000000000281526014018b73ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff166c010000000000000000000000000281526014018a73ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff166c010000000000000000000000000281526014018973ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff166c010000000000000000000000000281526014018873ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff166c010000000000000000000000000281526014018781526020018681526020018581526020018481526020018381526020018281526020019c5050505050505050505050505060405180910390209050809050929150505600a165627a7a723058206b1ebdd17a67bf269db5a531a541211ee288b29a5b08bba46ebabe6178531f580029",
+            "updated_at": 1518116202460
         }
     }
-}
+}
```

Comment:
These do not pass prettier as-is. If you run prettier they will be fixed automatically.